source/monitoring.rst

```diff
@@ -0,0 +1,113 @@
+.. _operator.monitoring:
+
+Monitoring
+==========
+
+Percona Monitoring and Management (PMM) `provides an excellent
+solution <https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html>`_
+of monitoring Percona Server for MongoDB.
+
+.. note:: Only PMM 2.x versions are supported by the Operator.
+
+PMM is a client/server application. *PMM Client* runs on each node with the
+database you wish to monitor: it collects needed metrics and sends gathered data
+to *PMM Server*. As a user, you connect to PMM Server to see database metrics on
+a number of dashboards.
+
+That's why PMM Server and PMM Client need to be installed separately.
+
+Installing PMM Server
+-------------------------
+
+PMM Server runs as a *Docker image*, a *virtual appliance*, or on an *AWS instance*.
+Please refer to the `official PMM documentation <https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/index.html>`_
+for the installation instructions.
+
+Installing PMM Client
+-------------------------
+
+The following steps are needed for the PMM client installation in your
+Kubernetes-based environment:
+
+#. The PMM client installation is initiated by updating the ``pmm``
+   section in the
+   `deploy/cr.yaml <https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/cr.yaml>`_
+   file.
+
+   -  set ``pmm.enabled=true``
+   -  ensure the ``serverHost`` (the PMM service name is
+      ``monitoring-service`` by default) is the same as the value specified
+      for the ``name`` parameter on the previous step, but with an added
+      additional ``-service`` suffix.
```

Comment:
it was actual if PMM installed inside kubernetes, now it is just "hostname" of  PMM Server